src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -20,6 +20,10 @@
  * Config manager for the robot.
  */
 public class Config {
+    static {
```

Comment:
Is it necessary to call init twice?

Reply:
In case the developer forgets to call `Config.init()` it makes sense. Inside the method itself, it checks whether or not it's been initialized already and won't do anything if it has.